To eliminate an rpmlint error, the shebang for this script should be
changed to use python directly, rather than through env.

When openSUSE switches to Python3, the shebang below should be changed
to "#!/usr/bin/python3".

Larry Finger

Index: VirtualBox-6.1.14/src/VBox/Frontends/VBoxShell/vboxshell.py
===================================================================
--- VirtualBox-6.1.14.orig/src/VBox/Frontends/VBoxShell/vboxshell.py
+++ VirtualBox-6.1.14/src/VBox/Frontends/VBoxShell/vboxshell.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python3
 # -*- coding: utf-8 -*-
 # $Id: vboxshell.py $
 
